spec/create_release_backmerge_pull_request_spec.rb

```diff
@@ -0,0 +1,329 @@
+require 'spec_helper'
+
+describe Fastlane::Actions::CreateReleaseBackmergePullRequestAction do
+  let(:test_token) { 'ghp_fake_token' }
+  let(:test_repo) { 'repo-test/project-test' }
+  let(:default_branch) { 'main' }
+  let(:other_action_mock) { double }
+  let(:client) do
+    instance_double(
+      Octokit::Client,
+      user: instance_double('User', name: 'test'),
+      list_milestones: %w[30.7 30.6 30.5 30.5.1 30.4].map { |version| mock_milestone(version) },
+      'auto_paginate=': nil
+    )
+  end
+
+  def mock_milestone(release_branch)
+    release_version = release_branch.delete('release/')
+    number = release_version.gsub('.', '').to_i
+    sawyer_resource_stub(title: release_version, number: number)
+  end
+
+  def mock_pr_url(release_branch)
+    number = release_branch.delete('release/').gsub('.', '').to_i
+    "https://github.com/#{test_repo}/pull/#{number}"
+  end
+
+  def stub_git_release_branches(branches)
+    allow(Fastlane::Actions).to receive(:sh)
+      .with('git', 'branch', '-r', '-l', 'origin/release/*')
+      .and_return("\n" + branches.map { |release| "origin/#{release}" }.join("\n") + "\n")
+  end
+
+  def stub_expected_pull_requests(expected_backmerge_branches:, source_branch:, labels: [], milestone_number: nil, reviewers: nil, team_reviewers: nil, commits_between_head_and_base: 42)
```

Comment:
💄  Just for the sake of conciseness, what about naming that param `nb_new_commits` instead of `commits_between_head_and_base`? 😄